config/config.go

```diff
@@ -110,16 +109,6 @@ type options struct {
 	rt http.RoundTripper
 }
 
-var globalConfigDhtTarget krpc.ID
-
-func init() {
```

Comment:
why was this removed?

Reply:
We weren't actually using it, and we just overlooked removing it.